vou/person.py

```diff
@@ -163,8 +163,8 @@ def did_overdose(self, x0: float = 1243.6936832876, k: float = 0.0143710866):
         roughly equals 2. We define "excess" as (dose / tolerance) - 1, or roughly 1 at
         steady state. We multiply the person's baseline OD risk by this excess squared.
         """
-        dose = self.concentration[-1]
-        tolerance = self.habit[-1]
+        dose = self.concentration[-2]
+        tolerance = self.habit[-2]
```

Comment:
I think I follow the logic on why we're going to two steps back.  My question is, would we only want to look two steps back if it's a pill and not injection/other (where we'd want to look one step back)? This could be totally off base, so ignore me if I'm totally wrong lol

Reply:
Added a comment.